spec/index.html

```diff
@@ -674,8 +694,9 @@ <h2>Literals</h2>
       elements:</p>
 
     <ul>
-      <li>a <dfn>lexical form</dfn>, being a Unicode [[!UNICODE]] string,
-        which SHOULD be in Normal Form&nbsp;C [[!NFC]],</li>
+      <li>a <dfn>lexical form</dfn> consisting of a sequence of
+        <a data-lt="code point" class="lint-ignore">Unicode code points</a> [[!UNICODE]]
+        matching the <a data-cite="XML10#charsets">Char</a> production [[XML10]].</li>
```

Comment:
Concepts already uses XML 1.1.   This reference needs to be updated.

Reply:
There are a couple of specs which still reference XML10, that need to be updated to XML11.